### DISCUSSION TITLE
Mac Metal support

### DISCUSSION THREAD

**Discussion by slavag** [ParisNeo/lollms-webui#314] [General]
Hi,
llama.cpp and llama-cpp-python are supporting Mac metal for sometime already, is there any plans to add support to metal to the lollms ?

Thanks

**Comment by d13g4** [comment]
Shouldn't that already work with the llama-cpp binding (from the binding-zoo)?

**Comment by ParisNeo** [comment]
Hi there, basically all bindings today support metal. If you installed lollms ages ago (few months), just upgrade it or reinstall the latest versions and you'll get the support.
